tests/python/test_solana.py

```diff
@@ -1,65 +1,36 @@
-from .apps.solana_utils import enable_blind_signing, enable_short_public_key, enable_expert_mode, navigation_helper_confirm, navigation_helper_reject
+from .apps.solana_utils import enable_blind_signing, enable_expert_mode
 
 
 class TestGetPublicKey:
 
-    def test_solana_get_public_key_ok(self, backend, navigator, test_name):
+    def test_solana_get_public_key_ok(self, backend, scenario_navigator):
         sol = SolanaClient(backend)
         from_public_key = sol.get_public_key(SOL_PACKED_DERIVATION_PATH)
 
         with sol.send_public_key_with_confirm(SOL_PACKED_DERIVATION_PATH):
-            if backend.firmware.device.startswith("nano"):
-                navigator.navigate_until_text_and_compare(NavInsID.RIGHT_CLICK,
-                                                          [NavInsID.BOTH_CLICK],
-                                                          "Approve",
-                                                          ROOT_SCREENSHOT_PATH,
-                                                          test_name)
-            else:
-                instructions = [
-                    NavInsID.USE_CASE_REVIEW_TAP,
-                    NavInsID.USE_CASE_ADDRESS_CONFIRMATION_CONFIRM,
-                    NavInsID.USE_CASE_STATUS_DISMISS
-                ]
-                navigator.navigate_and_compare(ROOT_SCREENSHOT_PATH,
-                                               test_name,
-                                               instructions)
+            scenario_navigator.address_review_approve(path=ROOT_SCREENSHOT_PATH)
 
         assert sol.get_async_response().data == from_public_key
 
 
-    def test_solana_get_public_key_refused(self, backend, navigator, test_name):
+    def test_solana_get_public_key_refused(self, backend, scenario_navigator):
         sol = SolanaClient(backend)
         with sol.send_public_key_with_confirm(SOL_PACKED_DERIVATION_PATH):
             backend.raise_policy = RaisePolicy.RAISE_NOTHING
-            if backend.firmware.device.startswith("nano"):
-                navigator.navigate_until_text_and_compare(NavInsID.RIGHT_CLICK,
-                                                          [NavInsID.BOTH_CLICK],
-                                                          "Reject",
-                                                          ROOT_SCREENSHOT_PATH,
-                                                          test_name)
-            else:
-                instructions = [
-                    NavInsID.USE_CASE_REVIEW_TAP,
-                    NavInsID.USE_CASE_ADDRESS_CONFIRMATION_CANCEL,
-                    NavInsID.USE_CASE_STATUS_DISMISS
-                ]
-                navigator.navigate_and_compare(ROOT_SCREENSHOT_PATH,
-                                               test_name,
-                                               instructions)
+            scenario_navigator.address_review_reject(path=ROOT_SCREENSHOT_PATH)
```

Comment:
Probably the `path` is the default one, tus not needed to give it as a parameter :thinking:

Reply:
It's required because the path of the tests is not the usual (there's an additional `python` dir in the `tests` dir...

If I remove it I get : `tests/python/test_solana.py::TestGetPublicKey::test_solana_get_public_key_refused[stax] - ValueError: Golden snapshots directory does not exist.`